internal/pkg/flink/internal/store/store_test.go

```diff
@@ -363,32 +363,66 @@ func (s *StoreTestSuite) TestParseSETStatement() {
 	key, value, _ = parseSetStatement("set 'key= \nvalue'	")
 	assert.Equal(s.T(), "", key)
 	assert.Equal(s.T(), "", value)
+
+	key, value, _ = parseSetStatement(`set ''key''=''value''`)
```

Comment:
This should error, right? Shouldn't we be testing that there's an error, instead of testing that the strings are empty?

Reply:
What if I want a single quote in my SET statement? Do I have to backslash it?

Reply:
> You would use a double quote inside the key/value string if you want to have a quote in there.

Sounds like there's no way to have a single quote inside the key/value string, though.

Reply:
Yeah, this wouldn't be allowed. I'll double check tomorrow if there's a way to do it with the open source client!

Reply:
Okay you can escape a single quote now with another single quote, that's how you can do it in the OS client as well!